zipline/data/adjustments.py

```diff
@@ -0,0 +1,599 @@
+from collections import namedtuple
```

Comment:
do I need to look over any particular changes in here or was this purely moved?

Reply:
This was just moved.